### PR TITLE
[ADD] accounting: provide information on the tax carryover feature

### PR DESCRIPTION
**Pull request by Donapi**
[ID: 2523121](https://www.odoo.com/web#id=2523121&cids=1&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/finance/accounting/reporting/overview/tax_carryover.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+=============
+Tax carryover
+=============
+
+When performing tax reports, the :guilabel:`tax carryover` feature allows carrying amounts from one
+period to the other without creating new entries.
+
+It has been created to respond to legal requirements in some locations where amounts must be
+transferred from period to period (for example, because the total of the line is negative).
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/overview/tax_carryover.rst:9]: It has been created to meet the legal requirements of specific locations, where amounts must be transferred from period to period (for instance, when the total of a line is negative).

File: content/applications/finance/accounting/reporting/overview/tax_carryover.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The feature is available per default in countries where it is required, such as Belgium, France, or
+Italy. There is no specific configuration required.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/overview/tax_carryover.rst:12]: The feature is activated by default in countries where it is required, such as Belgium, France, and
Italy. There is no specific configuration required.

File: content/applications/finance/accounting/reporting/overview/tax_carryover.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+In this case, as per local regulation, grid 81 of the tax report may contain a negative amount. But
+it must be declared to the government as zero, and the negative amount should be carried over to the
+next period.
+
+Go to :menuselection:`Accounting --> Reporting --> Tax Report`. You can see that a pop-up on line 81
+explains that the amount will be carried over in the next period.
+
+.. image:: tax_carryover/pop-up.png
+   :align: center
+   :alt: pop-up message stating the amount will be carried over to the next period
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/overview/tax_carryover.rst:30]: Not sure this last image is compressed , also not sure about the previous ones. As well as not sure if the zoom is 100% --> should be 110%

File: content/applications/finance/accounting/reporting/overview/tax_carryover.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+At the time of tax closing period, the tax report shows that the amount was carried over from the
+previous period. It also indicates the amount that will be carried over to this line in the next
+period based on the existing transactions and the carryover from the previous period.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/reporting/overview/tax_carryover.rst:34]: At the time of the tax closing period, the tax report shows that the amount was carried over from the
previous period. It also indicates the amount that will be carried over to this line in the next
period based on the existing transactions and the carryover from the previous period.

**Review by chiaraprattico — CHANGES_REQUESTED** [review]

**Review by chiaraprattico — APPROVED** [review]

File: content/applications/finance/accounting/reporting/overview/tax_carryover.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+   :alt: Illustration with a credit note
+
+In this case, as per local regulation, grid 81 of the tax report may contain a negative amount. But
+it must be declared to the government as zero, and the negative amount should be carried over to the
+next period.
+
+Go to :menuselection:`Accounting --> Reporting --> Tax Report`. You can see that a pop-up on line 81
+explains that the amount will be carried over in the next period.
+
+.. image:: tax_carryover/pop-up.png
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/reporting/overview/tax_carryover.rst:28]: on my side it looks ok... last time you told me my screenshots were too small :D we can check together at the office if needed!

File: content/applications/finance/accounting/reporting/overview/tax_carryover.rst [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+   :align: center
+   :alt: pop-up message stating the amount will be carried over to the next period
+
+At the time of the tax closing period, the tax report shows that the amount was carried over from
+the previous period. It also indicates the amount that will be carried over to this line in the next
+period based on the existing transactions and the carryover from the previous period.
+
+.. image:: tax_carryover/tax-return.png
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/accounting/reporting/overview/tax_carryover.rst:36]: same feedback than before :)

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**